### PR TITLE
Added new exercise: Knapsack

### LINKED ISSUE #516: New exercise: Knapsack
I would like to implement the [knapsack](https://github.com/exercism/problem-specifications/tree/master/exercises/knapsack) exercise in this track.

### PR DESCRIPTION
**Pull request by lpizzinidev**
closes #516

### CODE REVIEW

**Comment by SaschaMann** [comment]
@exercism/reviewers

File: exercises/practice/knapsack/src/main/kotlin/Knapsack.kt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+data class Item(val weight: Int, val value: Int)
+
+fun knapsack(maximumWeight: Int, items: List<Item>): Int {
+    TODO("Implement this function to complete the task")
+}
```
Comment (kotp) [inline exercises/practice/knapsack/src/main/kotlin/Knapsack.kt:5]: Missing EOL at what would be line 5.

**Comment by kotp** [comment]
Other than that, it seems reasonable for someone that does not do Kotlin.  I do wonder if the jar file is something that is generated, and so should not be stored in the repository.

**Review by SleeplessByte — APPROVED** [review]

**Comment by SleeplessByte** [comment]
@kotp the jar is indeed generated, but it's normal/common practice to commit it.

**Pull request merged**